.travis.yml

```diff
@@ -39,6 +39,7 @@ install:
 
   # Install drupal default profile and enable required modules.
   - /usr/bin/env PHP_OPTIONS="-d sendmail_path=`which true`" drush --verbose site-install --db-url=mysql://root:@127.0.0.1/drupal --yes
+  - composer install --dev
```

Comment:
The branch [failed on travis out of blue](https://travis-ci.org/acquia/acquia_lift/builds/176609525). Adding this line resolved the issue. I think we should keep it.